kodilla-exeption/src/main/java/com/kodilla/exception/test/FlightSearch.java

```diff
@@ -0,0 +1,34 @@
+package com.kodilla.exception.test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class FlightSearch {
+    public static void findFlight(Flight flight)throws RouteNotFoundException{
+        Map<String , Boolean> airports = new HashMap<>();
+        airports.put("GDI", true);
+        airports.put("WAW", false);
+        try {
+            if(airports.get(flight.getArrivalAirport())){
+                System.out.println("You can fly from: " + flight.getDepartureAirport()+" to: " + flight.getArrivalAirport());
+            }
+            else {
+                System.out.println("You can not fly from: " + flight.getDepartureAirport() + " to: " + flight.getArrivalAirport());
+                throw new RouteNotFoundException();
```

Comment:
RouteNotFoundException can store some additional data: in this case it will be important to store information which route cannot be found. So constructor should take as an parameter flight, and inside constructor you should create exception message (use this constructor: https://docs.oracle.com/javase/7/docs/api/java/lang/Exception.html#Exception(java.lang.String))